lib/ex_fizz_buzz_enterprise/input_state.ex

```diff
@@ -0,0 +1,15 @@
+defmodule ExFizzBuzzEnterprise.InputState do
+  defstruct [start_num: 0, current_num: 0, end_num: 100]
```

Comment:
It's not necessary to use the `[]` here.  In general, if a `Keyword.t` is the last argument to a function, the square brackets may be omitted.  I.e.,  `foo(1, 2, [bar: 42, baz: "hullo"])` is equivalent to `foo(1, 2, bar: 42, baz: "hullo")`.  So `defstruct [start_num: 0, current_num: 0, end_num: 100]` is equivalent to `defstruct([start_num: 0, current_num: 0, end_num: 100])` is equivalent to `defstruct(start_num: 0, current_num: 0, end_num: 100)` is equivalent to `defstruct  start_num: 0, current_num: 0, end_num: 100`.  I often opt for the `defstruct(start_enum: 0, current_enum: 0, end_enum:100)` variation just because my editor seems to like that better when it becomes

```
  defstruct(start_num: 0,
                 current_num: 0,
                 end_num: 100)
```